ci-operator/config/openstack-k8s-operators/barbican-operator/openstack-k8s-operators-barbican-operator-master.yaml

```diff
@@ -0,0 +1,22 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
```

Comment:
In other operators, we run unit as part of pre-commit
If you have functional tests, lets add them here instead:
```
- as: functional
  commands: |
    mkdir -p ../operator && cp -r . ../operator
    cd ../operator
    export GOFLAGS=
    make test GINKGO_ARGS='--no-color'
  container:
    from: src
```